frontend/src/components/Admin/AddUser/AddUser.tsx

```diff
@@ -125,6 +129,75 @@ export default function AddUser(): JSX.Element {
     });
   }
 
+  async function processJson(json: any[]): Promise<void> {
+    json.forEach((m) => {
+      const netId = getNetIDFromEmail(m.email);
+      const currMember = allMembers.find((mem) => mem.netid === netId);
+      if (currMember) {
+        const updatedMember = {
+          netid: netId,
+          email: m.email,
+          firstName: m.firstName || currMember.firstName,
+          lastName: m.lastName || currMember.lastName,
+          pronouns: m.pronouns || currMember.pronouns,
+          graduation: m.graduation || currMember.graduation,
+          major: m.major || currMember.major,
+          doubleMajor: m.doubleMajor || currMember.doubleMajor,
+          minor: m.minor || currMember.minor,
+          website: m.website || currMember.website,
+          linkedin: m.linkedin || currMember.linkedin,
+          github: m.github || currMember.github,
+          hometown: m.hometown || currMember.hometown,
+          about: m.about || currMember.about,
+          subteams: [m.subteam] || currMember.subteams,
```

Comment:
Empty arrays are truthy in JavaScript! If you test this out with a csv and you don't specify the subteam, the existing subteams will actually be deleted. Do `m.subteam ? [m.subteam] : currMember.subteams`

Reply:
<img width="961" alt="Screenshot 2023-09-19 at 10 20 56 PM" src="https://github.com/cornell-dti/idol/assets/59291082/54851579-9c91-442e-a47a-b85841020e4f">